src/main/java/seedu/address/model/pet/Color.java

```diff
@@ -6,40 +6,40 @@
  * Abstracts the color of a pet.
  */
 public class Color {
-    private final String color;
+    private final String value;
 
     /**
      * Constructs a color object.
      *
-     * @param color The string representation of a color.
+     * @param value The string representation of a color.
      */
-    public Color(String color) {
-        this.color = Objects.requireNonNullElse(color, "");
+    public Color(String value) {
```

Comment:
Can you update the method name in the Color test so that the test case will pass and CI will pass? thanks!